### PR TITLE
Tiddlywiki com

### PR DESCRIPTION
**Pull request by ibnishak**
Documentation updates involving Timimi, URL updates of few community resources

### CODE REVIEW

File: editions/tw5.com/tiddlers/nodejs/Serving TW5 from Android.tid [hunk 1]
```diff
@@ -1,13 +1,46 @@
...
+delivery: App with DIY steps
+description: Using Node.js to serve/create flatfile wikis
+method: sync
+modified: 20200501120801899
+tags: Saving [[TiddlyWiki on Node.js]] Android
 title: Serving TW5 from Android
 type: text/vnd.tiddlywiki
 
-[[Termux|https://termux.com/]] is an open source Android application that combines a Linux system and a terminal.
+[[Termux|https://termux.com/]] is and opensource android application providing limited unix environment enabling users to install [[Node.js]] and npm modules in android. Users can install and run [[TiddlyWiki on Node.js]] using [[Termux|https://termux.com/]].
```
Comment (Jermolene) [inline editions/tw5.com/tiddlers/nodejs/Serving TW5 from Android.tid:11]: I think "and opensource" is a misprint here? Open source is usually two words in English. And Unix should be capitalised.

Comment (ibnishak) [reply editions/tw5.com/tiddlers/nodejs/Serving TW5 from Android.tid]: Corrected

**Comment by ibnishak** [comment]
@Jermolene I have a workflow question.  If I have more documentation updates, should I push them to the same fork and you cherry-pick the commits or should I wait for you to merge, delete fork, fork anew and create another PR?

**Comment by Jermolene** [comment]
Hi @ibnishak apologies for the delay.

>  If I have more documentation updates, should I push them to the same fork and you cherry-pick the commits or should I wait for you to merge, delete fork, fork anew and create another PR?

Generally, smaller, self-contained PRs are easier to process, but that may not be practical if you're planning to make large scale contributions.

There are some general issues with the docs that I look for:

* Accuracy; a few times information that was wrong has crept into the docs which presents difficulties down the line
* Using the existing docs macros and conventions where they exist (e.g. filter docs involve a custom view template)

Beyond that, now that we have continuous integration set up, I'm usually able to merge docs updates relatively swiftly.

**Pull request merged**